src/components/layout/Navbar.vue

```diff
@@ -5,6 +5,7 @@
         <font-awesome-icon icon="bars" aria-hidden="true" v-show="!sidebar.hidden" />
       </a>
       <!-- AEGEE logo would be here -->
+      <p><a href="https://aegee.blogactiv.eu/2020/03/07/international-womens-day-2020-8th-of-march/">Read AEGEE's statement for the International Women's Day</a></p>
```

Comment:
@serge1peshcoff what about reserving this space with a component/variable text, that we can change more dynamically?

Reply:
sort of like creating a component that @mibaru-aegee created

Reply:
makes sense, but we should have a source for such things

Reply:
'source'? of info you mean? i could suggest the same redis we could use to keep the k/v of the static pages

Reply:
@linuxbandit yes, but redis won't do, we cannot connect to redis from the frontend directly

Reply:
why can't we use a js library that deals with redis in the `<script>` section? which doesn't even need to be performed client-side

Reply:
@linuxbandit https://stackoverflow.com/questions/6311984/connecting-directly-to-redis-with-client-side-javascript

anyway we can't allow some people to have a direct access to our db, it can go wrong in so many ways, that's what the backend is for :D

Reply:
ok I was unclear, we could do that and it would be a Very Bad Idea because duh -client-
In my very last sentence I actually pointed out that (for my limited understanding of oms-frontend) we do have some server-side js operations, couldn't we take advantage of that?

Reply:
We don't have server side things in browser, we have node (which is server side) to build stuff with webpack. So we need some kind of a wrapper for a db, most likely an API.

Not sure if I got your comment right, please clarify if not

Reply:
I think I understand my own confusion now, could it be that when/if we introduce SSR then what I am saying makes sense?

One step back: my goal is to make the map myaegee/confluence more dynamic than a commit (not necessarily real-time edit). This same data source could be then also used for this proposal i just made in this thread about use of a variable to fill the 'banner'

Reply:
@serge1peshcoff @linuxbandit I think this conversation might still be interesting, or not?

Reply:
i think yes, also on simpler terms: we could make some CSS time-dependant, so that every 8th march the colour changes. My proposal here is only a step forward to generalise things